api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -327,17 +330,19 @@ private String getUrlPath(Job job, String filePath) {
 
         Resource downloadResource = jobService.getResourceForJob(jobUuid, filename);
 
+        RequestAttributes requestAttributes = RequestContextHolder.getRequestAttributes();
```

Comment:
Are you using `requestAttributes` anywhere?

Reply:
Yes, it's used to get the response.